specs/NotifyDropTargetAction.md

```diff
@@ -0,0 +1,280 @@
+<!-- TEMPLATE
+  The purpose of this spec is to describe a new WebView2 feature and its APIs.
+
+  There are two audiences for the spec. The first are people
+  that want to evaluate and give feedback on the API, as part of
+  the submission process. When it's complete
+  it will be incorporated into the public documentation at
+  docs.microsoft.com (https://docs.microsoft.com/en-us/microsoft-edge/webview2/).
+  Hopefully we'll be able to copy it mostly verbatim.
+  So the second audience is everyone that reads there to learn how
+  and why to use this API. 
+-->
+
+
+# Background
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s)
+    in this spec. 
+
+    This section and the appendix are the only sections that likely
+    do not get copied into any official documentation, they're just an aid
+    to reading this spec. 
+    
+    If you're modifying an existing API, included a link here to the
+    existing page(s) or spec documentation.
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...". 
+-->
+
+This API will allow users to drop things such as images, text, and links into the WebView as part of a drag/drop operation.
+The reason that we need a separate API for this with composition hosting is because we don't have an HWND to call RegisterDragDrop on.
+The hosting app needs to call RegisterDragDrop on the HWND that contains the WebView and implement IDropTarget so it can forward the
+calls (IDropTarget::DragEnter, DragMove, DragLeave, and Drop) to the WebView.
+
+Additionally, the hosting app also needs to call into IDropTargetHelper before forwarding those calls to us as documented here:
+https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nn-shobjidl_core-idroptargethelper
+
+
+# Description
+<!-- TEMPLATE
+    Use this section to provide a brief description of the feature.
+
+    For an example, see the introduction to the PasswordBox control
+    (http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box).
+-->
+
+NotifyDropTargetAction is meant to provide a way for composition hosted WebViews to receive drop events as part of a drag/drop operation.
+It is the hosting application's responsibility to call RegisterDragDrop (https://docs.microsoft.com/en-us/windows/win32/api/ole2/nf-ole2-registerdragdrop)
+on the HWND that contains any composition hosted WebViews and to implement IDropTarget(https://docs.microsoft.com/en-us/windows/win32/api/oleidl/nn-oleidl-idroptarget)
+to receive the corresponding drop events from Ole32:
+
+IDropTarget::DragEnter
```

Comment:
Maybe this could be formatted like a list
* 1
* 2
* 3

Right now if you look at the doc preview instead of MD source, you see it all run together